AUTHORS.md

```diff
@@ -49,3 +49,12 @@ The following people have made contributions to this project:
 - [hazbottles (hazbottles)](https://github.com/hazbottles)
 - [oananicola (oananicola)](https://github.com/oananicola)
 - [praerien (praerien)](https://github.com/praerien)
+- [ralphk11 (ralphk11)](https://github.com/ralphk11)
+- [roquetp (roquetp)](https://github.com/roquetp)
+- [Sauli Joro (sjoro)](https://github.com/sjoro)
+- Guido della Bruna - meteoswiss
+- Marco Sassi - meteoswiss
+- [Rohan Daruwala (rdaruwala)](https://github.com/rdaruwala)
+- [Simon R. Proud (simonrp84)](https://github.com/simonrp84)
+- [Joleen Feltz (joleenf)](https://github.com/joleenf)
+- [Yufei Zhu (yufeizhu600)](https://github.com/yufeizhu600)
```

Comment:
Can you double check this list for me? The names that your PR have listed are already in the list with their real names. It looks like I/we didn't catch this in other PRs either because Oana is listed twice at least.